.github/CODEOWNERS

```diff
@@ -162,7 +162,7 @@
 /specification/monitor/ @gucalder
 
 # PRLabel: %Network
-/specification/network/ @GuptaVertika
+/specification/network/ @kamboj-prjwl
```

Comment:
Error(s) on line 165
Source Line: /specification/network/ @kamboj-prjwl
  -kamboj-prjwl is not a public member of Azure.

Reply:
What steps to follow to become public member of Azure.

Reply:
https://aka.ms/azsdk/access